### ISSUE TITLE
Alt Text should be seperated

### ISSUE THREAD

**Issue by JakeWags** [visdesignlab/upset2#264]
Alt text should be short. The larger version should be in a "long description". This could be through a link to a new tab. EPUB/daisy have the best interface for this

**Comment by JakeWags** [comment]
"For complete data information, navigate to the data table." or similar

**Comment by JakeWags** [comment]
Depends on [alt-txt #30](https://github.com/visdesignlab/upset-alt-txt-gen/issues/30)

**Comment by JakeWags** [comment]
Title
Caption
UpSet description
Alt Text
